src/lsm/tree.zig

```diff
@@ -580,6 +580,10 @@ pub fn TreeType(comptime TreeTable: type, comptime Storage: type, comptime tree_
             assert(tree.compaction_io_pending == 0);
             assert(tree.compaction_callback == null);
 
+            if (constants.verify) {
```

Comment:
This check, and the one below, may slow the fuzzer down too much to be worthwhile.